api/controller/user.js

```diff
@@ -12,3 +13,41 @@ export const getUserProfile = async (req, res) => {
     return res.status(500).json({ error: "Internal Server Error" });
   }
 };
+
+export const ContactAdmin = async (req, resp) => {
+  if (req.method !== "POST") {
+    return resp.status(405).send("Method Not Allowed");
+  }
+
+  const { Email, Subject, Message } = req.body;
+
+  const transporter = nodemailer.createTransport({
+    service: "gmail",
+    auth: {
+      user: `${process.env.Email}`,
+      pass: `${process.env.Key}`,
+    },
+  });
+  // user: `${process.env.Email}`,  add your email
+  // pass: `${process.env.Key}`,  add key which you will get  by enabling 2 step authentication from google for free
+
+  const mailOptions = {
+    from: Email,
+    to: "mustafizkaifee04@gmail.com",
+    subject: `Message From Globe Hoppin`,
+    text: `
+      Email: ${Email}      
+      Subject:${Subject}
+      Message: ${Message}`,
+  };
```

Comment:
Better create a class for notification, add method `sendEmail()` take arguments then call this method from here.
Later on we can extend that class for further use case.